feature-libs/product/configurators/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -0,0 +1,92 @@
+<div
+  class="cx-configurator-attribute-product-card"
+  [ngClass]="{
+    'cx-configurator-attribute-product-card-selected': product.selected
+  }"
+>
+  <div class="cx-configurator-attribute-product-card-rows">
+    <div class="cx-configurator-attribute-product-card-imgs">
+      <ng-container *ngIf="product?.images[0]; else noimages">
```

Comment:
check the length instead of first element maybe?

Reply:
changed